bioio_ome_zarr/reader.py

```diff
@@ -4,114 +4,127 @@
 from pathlib import Path
 from typing import Any, Dict, List, Optional, Tuple
 
+import dask.array as da
 import xarray as xr
-import zarr.storage
+import zarr
 from bioio_base import constants, dimensions, exceptions, io, reader, types
 from fsspec.spec import AbstractFileSystem
-from ome_zarr.io import ZarrLocation
-from ome_zarr.reader import Reader as ZarrReader
+from s3fs import S3FileSystem
```

Comment:
There's probably a way we don't need this since its just used for typing

Reply:
I think it's still worth it since it covers the various paths for S3 so that we don't have to handle that directly. If it becomes an issue/ depreciates then we can reconsider.